src/main/kotlin/pl/edu/agh/gem/external/persistence/MongoNotVerifiedUserRepository.kt

```diff
@@ -25,6 +25,10 @@ class MongoNotVerifiedUserRepository(
         return mongo.findOne(query, NotVerifiedUserEntity::class.java)?.toDomain()
     }
 
+    override fun delete(notVerifiedUser: NotVerifiedUser) {
```

Comment:
IMHO i change that to deleteById or sth like that in case we make some operation on domain object and want to delete primary entity